presto-docs/src/main/sphinx/sql/create-table-as.rst

```diff
@@ -33,7 +33,7 @@ Examples
 
 Create a new table ``orders_by_date`` that summarizes ``orders``::
 
-    CREATE TABLE orders_by_date
+    CREATE TABLE orders_by_date (order_date, sum_totalprice)
```

Comment:
You should probably leave the basic examples as they are and add some new one. Otherwise one could think the column aliases are required.